src/mixins/itext_click_behavior.mixin.js

```diff
@@ -152,7 +152,7 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
    */
   mouseUpHandler: function(options) {
     this.__isMousedown = false;
-    if (!this.editable || this.group ||
+    if (!this.editable ||
```

Comment:
this is a strong shortcut to remove.
For which reason this check is removed?
To make possible use a corner of an object in a group? or use the object at all?

Reply:
use at all.
it's the same as `skipCorners`, put here not to handle the case.

Reply:
Tested and looks good + #7598